### DISCUSSION TITLE
Weasis require administrator rights?

### DISCUSSION THREAD

**Discussion by vitriolvm** [nroduit/Weasis#447] [Q&A]
Hi all! First, Weasis si a fantastic project. Huge thanks to @nroduit and all the community for making a great product available to everyone.
Now, I need to deploy Weasis in many computer, some of which are Windows 10 and many others are still Windows 7. For W10 of course I deployed the very last 64 bit release, no issues there, but for the PCs running Windows 7 I have a problem. These systems run 32 bit OS, so I had to deploy the last compatible Weasis release for this architecture, which is 3.6.2: the problem is that **the executable of this version seems to require administrative rights to launch**.

Is this right? I didn't find any Q&A about that and nothing on a Google search came up. Did you guys have any clue if I can find a Weasis release for 32 bit systems that don't need administrator rights to launch the program?

Thanks in advance!

**Comment by nroduit** [comment]
For systems prior to Windows 10 they are already [end of life](https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions), it should no longer be used, same for [32-bit system](https://www.bleepingcomputer.com/news/microsoft/rip-microsoft-to-drop-support-for-windows-10-on-32-bit-systems/).

As it's already a lot of effort to get Weasis supported on many systems and architectures, it's obvious that end-of-life systems can't be supported. Another reason is that I don't think it's appropriate to have a 32-bit system to view exams that can be several GB because of memory limitations.

**Comment by vitriolvm** [comment]
Hi @nroduit. Thanks for the answer. I understand your points, and I'm for sure not asking for any fix on the code. Our 32 bit PCs are about to get upgraded and migrated to W11 64, we just need to provide the users with the viewer while we do that.

My question was just this: why do the executable needs administrative rights? Am I doing something wrong? Is there an older version that can be launched by restricted users?

Thanks again!

**Comment by nroduit** [comment]
As this distribution was discontinued 3 years ago, I can't remember. 

I remember that there were memory and installer problems, but not this one.
